spec/serializers/user_serializer_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe UserSerializer do
+  fab!(:user) { Fabricate(:user) }
+
+  subject { described_class.new(user, scope: guardian).as_json }
+
+  before do
+    SiteSetting.calendar_enabled = true
+    user.upsert_custom_fields(DiscourseCalendar::REGION_CUSTOM_FIELD => "uk")
+  end
+
+  context "as user" do
```

Comment:
I would clarify we're testing for another user.

```suggestion
  context "as another user" do
```